nsis/gvim.nsi

```diff
@@ -1081,6 +1081,12 @@ SectionGroup "un.$(str_ungroup_plugin)" id_ungroup_plugin
 		Pop $0
 
 		${If} $0 != ""
+		  ${If} ${FileExists} $0\_viminfo
+		    Delete $0\_viminfo
+		  ${EndIf}
+		  ${If} ${FileExists} $0\vimfiles\.netrwhist*
+		    Delete $0\vimfiles\.netrwhist*
+		  ${EndIf}
```

Comment:
Shouldn't the _vimfino and .netrwhist file added to the `RemoveVimfiles` macro?

Reply:
Yes, I also took a fresh look at the code yesterday and thought the same thing.
I'll redo it a little later.